### DISCUSSION TITLE
Issues with the Manjaro image

### DISCUSSION THREAD

**Discussion by builderjer** [OpenVoiceOS/ovos-buildroot#103] [Q&A] [answered]
To start off, I love Arch and use it as my primary distro. But, as with many Linux users, I have trouble with sound. I know that this is supposed to use pulseaudio, but I'm not sure how to configure it to work. I have not made any changes to the config files, as I don't know where to start. I have a USB speaker, and it shows up in pactl , but no output at all. Any ideas where to start would be great.

**Comment by Steve973** [comment]
Is the usbutils package installed?  Do you see it if you type "lsusb"?  If usbutils is not installed, then that would be your first step.

**Reply by builderjer** [reply]
This has been resolved.  I am a couple images past this one now.  Thanks

**Comment by builderjer** [comment, marked as answer]
This has been resolved with newer images
